### PR TITLE
NER tutorial - notebook

### PR DESCRIPTION
**Pull request by ekmb**
1. NER tutorial - colab notebook is added
2. Inference from a file - for the final evaluation or to perform error analysis added
3. Online inference for a list of queries added 
4. Script to convert from iob format to NeMo accepted format added
5. Updated ner config to make it more general
6. Cleaned up of the p&c tutorial to make the notebooks look alike
7. Confusion matrix plot functions moved from Nemo 0.11 

TODO:
- [x] updated the notebook once .nemo file and the data is available on aws

### CODE REVIEW

**Comment by okuchaiev** [comment]
"Mary leaves in Santa Clara and works at NVIDIA" => "Mary lives in Santa Clara and works at NVIDIA"

File: examples/nlp/token_classification/data/import_from_iob_format.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    """
+    Convert data from the IOB format to NeMo accepted format described below.
+    in_file should be in the IOB format, see example here:
+    https://www.clips.uantwerpen.be/conll2003/ner/.
+
+    Args:
+        in_file: input file name
+        out_text_f: output file with text
+        out_labels_f: output file with labels
+        max_length:
```
Comment (yzhang123) [inline examples/nlp/token_classification/data/import_from_iob_format.py:31]: add comment

File: examples/nlp/token_classification/data/import_from_iob_format.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+                label = line.strip().split()[-1]
+                words.append(word)
+                labels.append(label)
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        description='Convert data from IOB format to the format compatible with nlp/examples/token_classification.py'
+    )
+    parser.add_argument("--data_file", required=True, type=str)
```
Comment (yzhang123) [inline examples/nlp/token_classification/data/import_from_iob_format.py:101]: add help

Comment (ekmb) [reply examples/nlp/token_classification/data/import_from_iob_format.py]: done

File: examples/nlp/token_classification/data/import_from_iob_format.py (hunk 1 above)
Comment (ekmb) [reply examples/nlp/token_classification/data/import_from_iob_format.py]: done

**Pull request merged**